plugins/inputs/fibaro/fibaro.go

```diff
@@ -69,11 +69,12 @@ type Devices struct {
 	Type       string `json:"type"`
 	Enabled    bool   `json:"enabled"`
 	Properties struct {
-		Dead   interface{} `json:"dead"`
-		Energy interface{} `json:"energy"`
-		Power  interface{} `json:"power"`
-		Value  interface{} `json:"value"`
-		Value2 interface{} `json:"value2"`
+		Dead    interface{} `json:"dead"`
+		Energy  interface{} `json:"energy"`
+		Power   interface{} `json:"power"`
+		Value   interface{} `json:"value"`
+		Value2  interface{} `json:"value2"`
+		Battery interface{} `json:"batteryLevel"`
```

Comment:
if someone wants to pick this up later, this should probably be something more like this, and should check it against the published api. Be considerate of different api version out there in the wild.
```suggestion
		Dead    string `json:"dead"`
		Energy  string `json:"energy"`
		Power   string `json:"power"`
		Value   string `json:"value"`
		Value2  string `json:"value2"`
		Battery float64 `json:"batteryLevel"`
```